app/src/main/java/de/danoeh/antennapod/fragment/NewEpisodesFragment.java

```diff
@@ -49,7 +52,10 @@ protected boolean shouldUpdatedItemRemainInList(FeedItem item) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View root = super.onCreateViewHelper(inflater, container, savedInstanceState,
                 R.layout.all_episodes_fragment);
-
+        tvHeader = root.findViewById(R.id.emptyHeader);
+        tvHeader.setText(R.string.no_new_episodes_head_label);
+        tvEmpty = root.findViewById(R.id.empty);
+        tvEmpty.setText(R.string.no_new_episodes_label);
```

Comment:
There's no need to declare ```tvEmpty``` and ```tvHeader``` as variables since they are only being used once. You can simply do this:
```root.findViewById(R.id.emptyHeader).setText(R.string.no_new_episodes_head_label);```

Reply:
@iadeelzafar I have made the requested changes (https://github.com/AntennaPod/AntennaPod/pull/3060/commits/cabf6fee7b6e63ec9a77fdcf4bacea8c04abc335).